<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credits</class>
 <widget class="QWidget" name="Credits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="QScrollArea_Scroll">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="QWidget_ScrollContainer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>932</width>
        <height>686</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>35</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="QLabel_Explanation">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>SPlayer is a free and open-source media player software and recording utility developed by Saviz for playing video and audio as well as recording both. This project is availabel on the official GitHub repository of SPlayer. This project is licensed under &quot;License&quot; and free to be used on non-commercial products.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="QGroupBox_Information">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Application information</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>25</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Application Name:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="QLabel_Introduction_ApplicationName">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>name</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Application Version:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="QLabel_Introduction_ApplicationVersion">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>version</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Application Company:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="QLabel_Introduction_ApplicationCompany">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>company</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Application URL:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="QLabel_Introduction_ApplicationURL">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>url</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="QLabel_Introduction_OperatingSystem_Label">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Operating System:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="QLabel_Introduction_OperatingSystem">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>system</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="QVBoxLayout_Link">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Join the GitHub project and help us improve our product:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="QToolButton_Introduction_OpenPage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Open page</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>GNU GENERAL PUBLIC LICENSE
Version 3, 29 June 2007

Copyright (C) 2007 Free Software Foundation, Inc. &lt;https://fsf.org/&gt;
Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="QSpacer_Vertical">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
